.gitignore

```diff
@@ -121,4 +121,7 @@ venv.bak/
 *.swp
 *.swo
 
-torchtext/version.py
\ No newline at end of file
+torchtext/version.py
+
+# deleted files
+torchtext/.nfs*
```

Comment:
Is this from an editor?

Reply:
I removed this. I was using it to ignore some .nfs files that were polluting the directory